linters/pre-commit-hooks/plugin.yaml

```diff
@@ -5,6 +5,7 @@ lint:
       known_good_version: 4.4.0
       runtime: python
       package: pre-commit-hooks
+      description: A collection of git hooks for pre-commit
```

Comment:
I think we should tweak this since we don't use it in pre-commit, we run them always:
- A collection of linters to enforce style
- A collection of pre-commit tools for python